mender/values.yaml

```
@@ -475,6 +475,10 @@ auditlogs:
     annotations: {}
     type: ClusterIP
     port: 8080
+  env:
+    # Auditlog Expire Seconds gives the number of seconds before an
+    # audit event is evicted from the database
+    AUDITLOGS_AUDITLOG_EXPIRE_SECONDS: 2592000
```

Can we move this up one level and make it more obvious?
```suggestion
  logRetentionSeconds: 2592000 # 30days
```